lib/test/hip/RunGEMMKernelTileSelection_test.cpp

```diff
@@ -0,0 +1,253 @@
+/*******************************************************************************
```

Comment:
I don't see a difference between this and the test cases in RunGEMMKernel_test.cpp.  Could this be added to that as another test case?